doc/manual/general.rst

```diff
@@ -32,13 +32,18 @@ Introduction
 
 The SoundScape Renderer (SSR) is a software framework for real-time
 spatial audio reproduction running under GNU/Linux, Mac OS X and
-possibly some other UNIX variants. The current implementation provides
-Wave Field Synthesis (WFS), binaural (HRTF-based) reproduction, binaural
-room (re-)synthesis (BRTF-based reproduction), head-tracked binaural
-playback, Ambisonics Amplitude Panning (AAP), and Vector Base Amplitude
-Panning (VBAP). There are also the slightly exotic Generic Renderer. For
-each rendering algorithm there is a separate executable file. For more
-details see section :ref:`Renderers <renderers>`.
+possibly some other UNIX variants. The MS Windows version is experimental. 
+
+The current implementation provides:
+
+- Wave Field Synthesis (WFS)
```

Comment:
This is unrelated to this PR, but at some point we could provide a link for each of the renderers?
e.g. http://ssr.readthedocs.io/en/latest/renderers.html#wave-field-synthesis-renderer

I think this would be better than a single link to the "renderers" page in the next paragraph.

Reply:
I'll add this.